tools/deb/test/run_iptables_test.sh

```diff
@@ -6,11 +6,12 @@
 # Script requires a working docker on the test machine
 # It is run in the proxy dir, will create a docker image with proxy deb installed
 
```

Comment:
AFAIK that script is not run automatically, doesn't seem to pass - and is a different type of test.
This is small test - using golden files. We should have both kind of tests - I'm working on an
integration test that would include iptables (using a GCE machine).